example/src/routeBasedTemplate/index.html.js

```diff
@@ -0,0 +1,28 @@
+import head from '/partials/head'
+
+Index.routes = {
+  resolveIndex({ pathname }) {
```

Comment:
Is wat makkelijker als we het meteen als documentatie willen zien
```suggestion
  resolveIndex(location, req) {
    const { pathname } = location
```